### PR TITLE
feat: PURIS KIT Release 24.05

### PR DESCRIPTION
**Pull request by dominikoeh**
## Description

This contribution contains the content for the PURIS KIT release for the 24.05 release

It includes the following content

- Adoption View
- Software Development View

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

@danielmiehle @maximilianong @jSchuetz88

### CODE REVIEW

File: docs-kits/kits/PURIS Kit/Software Development View/page_software-development-view.md [hunk 1]
```diff
@@ -0,0 +1,727 @@
+---
```
Comment (HFocken) [inline docs-kits/kits/PURIS Kit/Software Development View/page_software-development-view.md:1]: General comment: please check the licenses for your pictures and include relevant info - please refer to the TRG 7.07

**Review by danielmiehle — DISMISSED** [review]
LGTM

File: docs-kits/kits/PURIS Kit/Software Development View/page_software-development-view.md [hunk 2]
```diff
@@ -0,0 +1,727 @@
...
+  "odrl:operator": {
+    "@id": "odrl:eq"
+  },
+  "odrl:rightOperand": "active"
+}
+```
+
+#### Framework Agreement Policy​
+
+To verify if a participant accepted the framework agreement of a specific use case created by the [Catena-X association](https://catena-x.net/en/about-us/the-association), the `FrameworkAgreement.puris` verifiable credential can be used for the PURIS framework agreement. In case of a policy, the data can only be used from accepted and verified PURIS framework agreement members. This is shown in the second constraint-part of the example above.
```
Comment (HFocken) [inline docs-kits/kits/PURIS Kit/Software Development View/page_software-development-view.md:132]: The more specific link to the framework conditions is here: https://catena-x.net/en/catena-x-introduce-implement/governance-framework-for-data-space-operations - please update

**Review by HFocken — CHANGES_REQUESTED** [review]
Thank you for the nice update - two minor comments / edits necessary, otherwise great!

File: docs-kits/kits/PURIS Kit/Software Development View/page_software-development-view.md (hunk 2 above)
Comment (dominikoeh) [reply docs-kits/kits/PURIS Kit/Software Development View/page_software-development-view.md]: Link has been updated

File: docs-kits/kits/PURIS Kit/Software Development View/page_software-development-view.md (hunk 1 above)
Comment (dominikoeh) [reply docs-kits/kits/PURIS Kit/Software Development View/page_software-development-view.md:1]: Licenses have been checked by the team

**Review by HFocken — APPROVED** [review]
Thanks for addressing the outstanding issues - good to go from my end!

**Review by jSchuetz88 — APPROVED** [review]
Thank you for the update, no comments from my end.

**Review by danielmiehle — APPROVED** [review]
LGTM

**Review by maximilianong — APPROVED** [review]

**Pull request merged**